src/index.js

```diff
@@ -0,0 +1,60 @@
+const writer = require('.\\writer.js');
+import '.\\styles.scss'
+
+//creating fieldSelector block and its contents
+document.body.append(writer.createHtml('div', 'fieldSelectorDiv flex'));
```

Comment:
I don't see MVC/MVVM pattern here. 

```
изначально body в файле index.html должен быть пустой, все необходимые элементы генерируются с использованием JS. Использовать MVC/MVVM паттерн
```

Reply:
I've done some changes. Please, check out if that is what u asked